updates/28/updates.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "Monerium EURe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "A", 100, "0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", 18, "None"); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x420CA0f9B9b604cE0fd9C18EF134C705e5Fa3430", "EURe", "monerium-eur-money", "EURE", NULL, 1724565650, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x8E34bfEC4f6Eb781f9743D9b4af99CD23F9b7053", "Monerium GBPe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x8E34bfEC4f6Eb781f9743D9b4af99CD23F9b7053", "A", 100, "0x8E34bfEC4f6Eb781f9743D9b4af99CD23F9b7053", 18, "None"); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x8E34bfEC4f6Eb781f9743D9b4af99CD23F9b7053", "GBPe", "monerium-gbp-emoney", "", NULL, 1724565650, NULL);
```

Comment:
also, should we fill the `swapped_for` field for these? I'm not sure because you were mentioning some problems with the way these tokens are updated.